services/api/src/resources/task/resolvers.js

```diff
@@ -347,6 +347,40 @@ const taskDrushCacheClear = async (
   return taskData;
 };
 
+const taskDrushCron = async (
+  root,
+  { environment: environmentId },
+  { sqlClient, hasPermission },
+) => {
+  await envValidators(sqlClient).environmentExists(environmentId);
+  await envValidators(sqlClient).environmentHasService(environmentId, 'cli');
+  const envPerm = await environmentHelpers(sqlClient).getEnvironmentById(environmentId);
+  await hasPermission('task', `drushCron:${envPerm.environmentType}`, {
+    project: envPerm.project,
+  });
+
+  const command =
+    'drupal_version=$(drush status drupal-version --format=list) && \
+  if [ ${drupal_version%.*.*} == "8" ]; then \
+    drush core:cron; \
+  elif [ ${drupal_version%.*} == "7" ]; then \
+    drush cc core-cron; \
+  else \
+    echo "could not execute cron for found Drupal Version ${drupal_version}"; \
+    exit 1; \
+  fi';
```

Comment:
Curious as to why you are running 2 different commands based on Drupal version. I thought `drush cron` is fairly ubiquitous.

Reply:
`dush cron` is an alias for `core-cron` (D7) and `core:cron` (D8).
If you aren't bothered I will change them to use the alias `cron`.

Reply:
Done.

Reply:
Yeah,  the command `cron` is what everyone understands. Also it makes this code a little more future proof when Drupal 9 comes out.